### ISSUE TITLE
Missing translations and new parameters for Maximizer

### ISSUE THREAD

**Issue by Digitalone1** [wwmm/pulseeffects#523]
Hello, I followed instructions [here](https://github.com/wwmm/pulseeffects/wiki/Translating-PulseEffects#manually), but new sentences in Configurations List tab are not shown. What am I doing wrong?

Gain Reduction and Output Level parameters are missing for Maximizer plugin. Please, add them in the next release.

**Comment by wwmm** [comment]
Which configuration list? The one where we dump Pulseaudio config file?

**Comment by Digitalone1** [comment]
> Which configuration list? The one where we dump Pulseaudio config file?

This:

![Screenshot_20190603_005411](https://user-images.githubusercontent.com/25790525/58768357-58a24c80-859a-11e9-9843-044e6de2bfcc.png)

Don't know which is the name of this tab.

**Comment by wwmm** [comment]
Oh, now I understand. The instructions in the link you pasted above are meant to be used only when you add new labels or widgets to the interface. Not for translations.

Did you try to run `ninja install` to a custom output path? If you do no install it somewhere the translation file of the PE package installed in the system will be used instead.

**Comment by Digitalone1** [comment]
> Oh, now I understand. The instructions in the link you pasted above are meant to be used only when you add new labels or other widgets to the interface. Not for translations.
> 
> Did you try to run `ninja install` to a custom output path? If you do no install it somewhere the translation file of the PE package installed in the system will be used instead.

Yes, I did, but in pulseffects.pot and it_IT.po there aren't new labels, even after I run 
```
cd _build
ninja pulseeffects-pot
ninja pulseeffects-update-po
```

**Comment by wwmm** [comment]
That is exactly what has to happen. If no `English` word was added, removed or changed in the interface nothing has to change in the translation template. And as a result no new word to be translated will be added to the `*.po` files. The file `it_IT.po` should have been updated when I updated the translation template before doing the release. At least the pt_BR file was updated at this moment. Maybe something went wrong with it_IT. Are the strings shown if you remove it's file and execute the commands above again?

Or are you adding new widgets to the interface? But in this case new words should have been added to the translation template. Unless the widgts label are marked as non translatable.

**Comment by Digitalone1** [comment]
You're right. Don't know why, but the fork in my repository had old files. I was sure I deleted it and reforked a new one. Now I redeleted and reforked and I got the new file. I will do a pull request for new translations.

Do you confirm that running 
```
meson _build --prefix=/usr
env DESTDIR="$pkgdir" ninja -C _build install
```
will not conflict with installed upstream package or should I change prefix also?

**Comment by wwmm** [comment]
I think it will depend on the `$pkgdir` value. If it is `/usr` I would no run as root because there will be conflict. What I usually do is `meson _build --prefix=/tmp/pe` and then `ninja install`. This way there is no need to be root and there is no conflict because everything will be put under `/tmp`

**Comment by wwmm** [comment]
To understand because using `tmp` works take a look at here https://github.com/wwmm/pulseeffects/blob/561ab2893fea23268c915f22cfcb2a4c337eac1d/src/pulseeffects.cpp#L21 and here https://github.com/wwmm/pulseeffects/blob/561ab2893fea23268c915f22cfcb2a4c337eac1d/meson.build#L30. At build time meson defines the variable LOCALE_DIR with the install path in a `config.h` file. That is why a install is needed to test translations.

**Comment by Digitalone1** [comment]
Okay, so I will update Wiki with these tips later.

**Comment by wwmm** [comment]
Gain reduction is already being used https://github.com/wwmm/pulseeffects/blob/8e853108b3bdb0e61bc1ea13a0c6dc907ef4fcce/src/maximizer.cpp#L16. But for some reason I do not remember I named it just `Reduction`. I will rename it to `Gain Reduction`.

Only the output level is missing.

**Comment by wwmm** [comment]
I wonder if `output-level` is just measureing the output signal level. IF it is then there is no need to add it because we already have a level meter in the output. It would be redundant.

**Comment by Digitalone1** [comment]
Oh, I though they was input parameters. So it seems they are only output values.

**Comment by wwmm** [comment]
They are read-only. In the output of `gst-inspect-1.0 ladspa-zamaximx2-ladspa-so-zamaximx2` we have
```
  gain-reduction      : Gain Reduction
                        flags: readable, controllable
                        Float. Range:               0 -              40 Default:               0 

output-level        : Output Level
                        flags: readable, controllable
                        Float. Range:             -45 -               0 Default:               0
```
Input parameters have the `writable` flag

**Comment by wwmm** [comment]
I released `4.6.4` with the new name. As far as I can see we already have these two info in the ui.
